.openshift-ci/gcp/command.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Maybe rename this file. Maybe something like run-command-in-gcp-vm.sh? Maybe that is too long.

Reply:
Yeah, sounds a bit too long. I've put a comment into this file, is that be enough to clarify it's purpose instead of changing the name, what would you say?